inc/functions/common.php

```diff
@@ -238,90 +238,92 @@ function imagify_can_optimize_custom_folders() {
 	return $can;
 }
 
-/**
- * Simple helper to get some external URLs, like to the documentation.
- *
- * @since  1.6.12
- * @author Grégory Viguier
- *
- * @param  string $target     What we want.
- * @param  array  $query_args An array of query arguments.
- * @return string The URL.
- */
-function imagify_get_external_url( $target, $query_args = array() ) {
-	$site_url = IMAGIFY_SITE_DOMAIN . '/';
-	$app_url  = IMAGIFY_APP_DOMAIN . '/#/';
-
-	switch ( $target ) {
-		case 'plugin':
-			/* translators: Plugin URI of the plugin/theme */
-			$url = __( 'https://wordpress.org/plugins/imagify/', 'imagify' );
-			break;
+if ( ! function_exists( 'imagify_get_external_url' ) ) {
```

Comment:
what is the need of wrapping the function in this conditional?

Reply:
We should not be modifying the code to make the tests work, it would be good to look into how the code needs to be modified to be easily testable instead.